zng/docs/spec.md

```diff
@@ -467,59 +500,57 @@ i.e., via `\u{0a}` or `\x0a`.
 A line that begins with `#` is a control message and all other lines
 are values.
 
-### 3.1 ZNG Control Messages
+### 4.1 Control Messages
 
-ZNG control messages have one of four forms defined below.
+TZNG control messages have one of four forms defined below.
 
 Any line beginning with `#` that does not conform with the syntax described here
 is an error.
-When errors are encountered parsing ZNG, an implementation should return a
-corresponding error and allow ZNG parsing to proceed if desired.
+When errors are encountered parsing TZNG, an implementation should return a
+corresponding error and allow TZNG parsing to proceed if desired.
 
-### 3.1.1 Type Binding
+### 4.1.1 Type Binding
 
-A type binding has the following form:
+A TZNG type binding has the following form:
 ```
 #<type-tag>:<type-string>
 ```
 Here, `<type-tag>` is a string decimal integer and `<type-string>`
-is a string defining a type according to the ZNG type syntax creates a binding
-between the indicated tag and the indicated type.
+is a string defining a type according to the ZNG type syntax. They create
```

Comment:
Can someone help point to the right place in the spec for the "ZNG type syntax" it's referencing here? I should surely put a hyperlink here.

Reply:
This is referring to section 4.2 below.

Reply:
Thanks @henridf. Based on that, I've not only added a hyperlink but set the link text to "TZNG type grammar" so that way a reader familiar with a spec will know what it's referencing without even having to click on the link.